tests/lib/snaps/test-snapd-service/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: test-snapd-service
+version: 1.0
+apps:
+ test-snapd-service:
+  command: bin/start
+  daemon: simple
```

Comment:
Still just one space here. These are generally four as well (or might be, perhaps :) so they're more readable.

Merging now, though. We can fix that on the next pass.